characters/electro/raiden.md

```diff
@@ -75,7 +75,7 @@ The Eye can initiate one coordinated attack every 0.9s per party. Coordinated at
 | Elemental Burst DMG Bonus | 0.27% Per Energy | -- | -- | -- |
 | CD | 10 s | -- | -- | -- |  
 * The Eye does not snapshot/is dynamic.  
-* The coorinated attack can only occur on damaging strikes against enemies.  
+* The coorinated attack can only occur on a damaging strike, against enemies, that is capable of inflicting AOE damage.  
```

Comment:
I don't think this line sufficiently describes how it procs. It implies all strikes must be aoe to deal damage and i'm not sure that that's necessarily true.

Reply:
all strikes require to be coded as AOE for it, which is hard to describe in simple words, for example an arrow is able to do aoe even tho it's tiny

Reply:
I think while true, it is still misleading and can cause confusion and should be reworded. I'll leave this as ac omment and leave it to your discretion